zboss/Kconfig

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "ZBOSS stack configuration."
+
+if ZIGBEE
+
+# Add a menu option to switch between compiled libraries and source code.
+choice
+	prompt "ZBOSS library type"
+	default ZBOSS_LIBRARY_PRECOMPILED
+
+config ZBOSS_LIBRARY_PRECOMPILED
+	bool "Libraries"
+	help
+	  Use ZBOSS compiled libraries while building Zigbee application
+
+config ZBOSS_LIBRARY_SOURCES
+	bool "Source code"
+	help
+	  Use ZBOSS source code while building Zigbee application
+
+endchoice
```

Comment:
This should be adjusted based on the outcome of: `<insert link>`

Reply:
Moved to NCS platform repository.